### ISSUE TITLE
Outdated PHP Vulnerability 

### ISSUE THREAD

**Issue by Aznyi** [linuxserver/docker-swag#230]
The PHP in the SWAG image is outdated and is considered a Critical vulnerability.

**Description**

The version of PHP installed on the remote host is prior to 7.4.28. It is, therefore, affected by a vulnerability as referenced in the Version 7.4.28 advisory.

- In PHP versions 7.4.x below 7.4.28, 8.0.x below 8.0.16, and 8.1.x below 8.1.3, when using filter functions with FILTER_VALIDATE_FLOAT filter and min/max limits, if the filter fails, there is a possibility to trigger use of allocated memory after free, which can result it crashes, and potentially in overwrite of other memory chunks and RCE. This issue affects: code that uses FILTER_VALIDATE_FLOAT with min/max limits.
(CVE-2021-21708)

**Version Info**

(7.4.26 under X-Powered-By: PHP/7.4.26)
  Installed version : 7.4.26
  Fixed version     : 7.4.28

**Solution**

Upgrade to PHP version 7.4.28 or later.

This was verified using Nessus.

**Comment by aptalca** [comment]
Please report it upstream to alpine. We're pulling from their 3.14 repo.

Thanks

**Comment by Aznyi** [comment]
Hopefully I put this in the right spot, I reported it here: https://github.com/alpinelinux/docker-alpine/issues/249

**Comment by aptalca** [comment]
No, their repo is here: https://gitlab.alpinelinux.org/alpine/aports/-/issues

**Comment by nemchik** [comment]
We are now shipping php 8.0.18
https://github.com/linuxserver/docker-swag/blob/58c0c03187d900d8451f42ddfa17bf2af4f657af/package_versions.txt#L130
